packages/vite-plugin-angular/src/lib/angular-vite-plugin.ts

```diff
@@ -159,6 +159,9 @@ export function angular(options?: PluginOptions): Plugin[] {
 
     return {
       name: '@analogjs/vite-plugin-angular',
+      async watchChange() {
+        await buildAndAnalyze();
```

Comment:
I think it should be wrapped in a `isTest` block so it still only applies when running tests.

```suggestion
      async watchChange() {
        if (isTest) {
          await buildAndAnalyze();
        }
```